### DISCUSSION TITLE
find a person

### DISCUSSION THREAD

**Discussion by msedaghati** [community/community#42210] [New to GitHub] [answered]
Hi
I downloaded a code in GitHub and I want to ask some question about it. But I can not find the person how write it. 
His /Her GitHub profile name is solidharr_zhang. How can I find His/her Email?

thanks

**Comment by samharp** [comment, marked as answer]
Hi there,

It looks like this user does not have an email address publicly displayed on their profile, so it is unfortunately impossible for you to reach out to them in this type of way.

If you are really wanting/needing to get in contact with them, however, you could try creating a Pull Request or Issue in one of their repositories, saying that you would love to get in contact with them and leaving your contact information that way. Be wary though as this would leave your contact information publicly accessible to anyone that viewed that respective repository. You could also try searching for that username on other social media networks to see if they exist elsewhere?

Hopefully that helps, and good luck in getting ahold of them!

**Reply by morzamohammaf** [reply]
Hi sir  I am signe out from my GitHub account, please let me know how to acces my account again? Thank you sir

**Comment by msedaghati** [comment]
Hi

Thanks for your reply

Best regards

**Comment by python012** [comment]
You can raise an issue in the repository, that could be one way to contact the author

**Reply by msedaghati** [reply]
thanks

**Comment by jfullstackdev** [comment]
it's for sure that the author is there at the repo that you downloaded, so raise an issue and if they are interested, they will reply to you

**Reply by msedaghati** [reply]
thanks

**Comment by msedaghati** [comment]
Hi
I have a question about Root .
Can I use Root for Image Reconstruction?
